src/main/java/manufacturing/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,19 @@
+package manufacturing.dao;
+
+import java.util.List;
+import java.util.Optional;
+import manufacturing.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> getById(Long manufacturerId);
+
+    Manufacturer update(Manufacturer manufacturer);
+
+    boolean deleteById(Long manufacturerId);
+
+    boolean delete(Manufacturer manufacturer);
+
+    List<Manufacturer> getAllManufacturers();
```

Comment:
It's not a big deal, but I will to have methods grouped by their functionality... again it's up to you